src/Core/Content/Product/Cms/ProductSliderCmsElementResolver.php

```diff
@@ -31,9 +32,16 @@ class ProductSliderCmsElementResolver extends AbstractCmsElementResolver
     private const PRODUCT_SLIDER_ENTITY_FALLBACK = 'product-slider-entity-fallback';
     private const STATIC_SEARCH_KEY = 'product-slider';
     private const FALLBACK_LIMIT = 50;
+    private const PRODUCT_ASSOCIATIONS = [
+        'cover',
```

Comment:
'cover' can be removed.
See https://github.com/shopware/shopware/pull/5260/files/aab1539965768738b97aa03178987ec0420d3ca7#r1814389953